scipy/stats/tests/test_crosstab.py

```diff
@@ -0,0 +1,75 @@
+import numpy as np
+from numpy.testing import assert_array_equal
+from scipy.stats import crosstab
+
+
+def test_crosstab_basic():
```

Comment:
can we put all the tests into a `class Test_crosstab` class for pytest? makes it easier to organize and share test input/output as the tests cases evolve. Even if we don't strictly ***need*** it now, it is nice to have and doesn't break anything setup here.

Reply:
I had missed that this is within the file of tests specific to `crosstab` so no. need for the pytest class.